Makefile

```diff
@@ -33,19 +33,23 @@ build: ## Builds the docker-compose environment
 	@echo "==> Building docker image..."
 	@docker-compose $(docker_files) build
 	@echo "    [✓]\n"
-run-only: # Runs the docker environment
+run-only: ## Runs the docker environment
 	@docker-compose $(docker_files) up
-run: build # Runs & builds the docker environment
+run: build ## Runs & builds the docker environment
 	@docker-compose $(docker_files) up
-logs: # Shows live logs if the workers are running or logs from last running worker if they are not.
+test: ## Assuming the environment is already running, executes unit tests
```

Comment:
It'd be nice to make this requirement explicit rather than assuming it. Some options:
* Add a prompt like "make sure the worker is running. Proceed?" 
* Just run the worker directly (if it's already running it won't start a new one).
* Actually detect if the worker is running and show a meaningful error message if it's not.

Reply:
I chose the third option. Now, if the container isn't running, you get a nice error message `The containers are not running. Run 'make run' and try again.`